### DISCUSSION TITLE
Save to upgrade to paid Elasticsearch plans?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#9887] [Unsupported Versions]
Hello!

I'm wondering if it's save to upgrade to a paid Elasticsearch plan in order to use more advanced features or if I risk to break SO? Any experiences?

Cheers, Ben

**Comment by InfosecGoon** [comment]
It is safe to install a paid Elastic license on Security Onion - it won't affect the functioning of the platform. With that said, there are some Elastic components that we hide or disable in Kibana, so there might be some licensed items that are not available by default - if you run into this, please let us know here in Discussions so we can find a solution and take it into account for future development.
